src/features/ITGovernance/Admin/GRBReview/GRBReviewStatusCard/index.tsx

```diff
@@ -1,131 +1,200 @@
+      <GRBReviewStatusTag grbReviewStatus={GRBReviewStatus.SCHEDULED} />
+
+      {/* Meeting Details */}
+      {/* TODO: !!!! Replace the Standard GRB Review Status field and enum */}
+      {grbReviewAsyncStatus !==
+        SystemIntakeGRBReviewAsyncStatusType.COMPLETED && (
+        <span>
+          <h4 className="margin-0 margin-right-1 margin-top-2px margin-bottom-05">
+            {t('statusCard.grbMeeting')}
           </h4>
 
-          <Tag className="bg-white text-base-darker font-body-sm">
-            {t(`statusCard.grbReviewStatus.${grbReviewStatus}`)}
-          </Tag>
+          <div className="easi-body-large">
+            {formatDateUtc(grbDate, 'MM/dd/yyyy')}
+          </div>
+
+          {isITGovAdmin && (
+            <UswdsReactLink
+              to={`/it-governance/${grbReview.id}/dates`}
+              className="usa-button usa-button--outline margin-top-1"
+            >
+              {t('statusCard.changeMeetingDate')}
+            </UswdsReactLink>
+          )}
         </span>
+      )}
+    </div>
+  );
+
+  const AsyncCard = (
+    <div
+      className={classNames(
+        className,
+        'padding-3 radius-md',
+        renderBGColor(grbReviewAsyncStatus)
+      )}
+    >
+      <h3 className="margin-top-0 margin-bottom-2">
+        {t('statusCard.asyncHeading')}
+      </h3>
+
+      {/* Status Section */}
+      <GRBReviewStatusTag grbReviewStatus={grbReviewAsyncStatus} />
+
+      {/* Meeting Details */}
+      {grbReviewAsyncStatus !==
+        SystemIntakeGRBReviewAsyncStatusType.COMPLETED && (
+        <span>
+          <h4 className="margin-0 margin-right-1 margin-top-2px margin-bottom-05">
+            {t('statusCard.timeRemaining')}
+          </h4>
 
-        {/* Meeting Details */}
-        {grbReviewStatus !== GRBReviewStatus.COMPLETED && (
-          <span>
-            <h4 className="margin-0 margin-right-1 margin-top-2px margin-bottom-05">
-              {t('statusCard.grbMeeting')}
-            </h4>
-
-            <div className="easi-body-large">
-              {formatDateUtc(grbDate, 'MM/dd/yyyy')}
-            </div>
-
-            {isITGovAdmin && (
-              <UswdsReactLink
-                to="./dates"
-                className="usa-button usa-button--outline margin-top-1"
-              >
-                {t('statusCard.changeMeetingDate')}
-              </UswdsReactLink>
-            )}
+          <span className="display-flex margin-bottom-05">
+            <p className="easi-body-large margin-top-0 margin-bottom-05 margin-right-2">
+              {t('statusCard.countdown', {
+                days,
+                hours,
+                minutes
+              })}
+            </p>
+
+            <p className="easi-body-normal text-primary-dark margin-0 flex-align-self-center">
+              {t('statusCard.reviewEnds', {
+                date: formatDateUtc(grbReviewAsyncEndDate, 'MM/dd/yyyy')
+              })}
+            </p>
           </span>
```

Comment:
The review details should be stacked on mobile instead of displayed side by side.

Current:
<img width="327" alt="Screenshot 2025-03-31 at 9 48 43 AM" src="https://github.com/user-attachments/assets/6fef10ac-97fb-42f7-9801-6b7723e59112" />

Figma:
<img width="444" alt="Screenshot 2025-03-31 at 9 53 22 AM" src="https://github.com/user-attachments/assets/274cef40-2a19-4d02-a642-96086bbaf3ba" />